src/Types.ts

```diff
@@ -127,13 +127,20 @@ export interface Document {
 
   is_private: boolean;
 
+  collection_id: number;
```

Comment:
Does this mean that every document has to be part of a collection?

Reply:
No. collection_id is null for non-collection documents (any uploaded, added by users).

Reply:
Ah got it, thanks. 👍  In that case the type should reflect that `collection_id?: number`. I'll add that change later!